### DISCUSSION TITLE
Is the software name lowercase or uppercase?

### DISCUSSION THREAD

**Discussion by manning-ncsa** [astrophpeter/blast#190] [Q&A]
[The documentation](https://blast.readthedocs.io) consistently references this software as "blast" with a lowercase "b", capitalizing it only when it begins a sentence. Is this indeed how you want this software to be referenced in general, for example in scientific publications?

My unsolicited opinion is that it should always be capitalized, primarily because "blast" is already a standard English word.

@djones1040 @astrophpeter

**Comment by djones1040** [comment]
I tend to agree, though not sure if standard practice for software would be lowercase.  Have been writing the paper with capital B.  That said, I think Peter should decide

**Reply by djones1040** [reply]
in the TeX we can use \tt to distinguish

**Reply by manning-ncsa** [reply]
>I tend to agree, though not sure if standard practice for software would be lowercase.

There is no standard practice of lowercase software names. In my experience nearly all software products are capitalized when referencing them in a sentence, even if their trademark logo is lowercase.

**Reply by astrophpeter** [reply]
ok, happy to have capital B :)

**Reply by manning-ncsa** [reply]
See [this related pull request](https://github.com/astrophpeter/blast/pull/192).
